README.md

```diff
@@ -2,6 +2,8 @@
 
 dokku-docker-auto-volumes is a plugin for [dokku][dokku] that automatically persist docker volumes declared in the Dockerfile.
 
+/var/lib/dokku/volumes 
```

Comment:
I agree - that was a personal note to myself - telling me where to look for files. 

My attempts to use the plugin have failed, as I do now know how to define the volumes properly. 

That is what got me searching for "where" they should go, and how to deal with failures. 

How does one define the volumes to persist?